test/test_data/glibc.py

```diff
@@ -0,0 +1,11 @@
+mapping_test_data = [
+    {"product": "glibc", "version": "2.31", "version_strings": ["GLIBC 2.31"]}
```

Comment:
```suggest
    {"product": "glibc", "version": "2.31", "version_strings": ["GLIBC 2.31", "This file is part of the GNU C Library."]}
```
version_strings list must have contains_strings.  This is only passing here because of a bug in our test_scanner which I have fixed in my recent PR.

Reply:
@Niraj-Kamdar  Hey Niraj. I just committed the necessary changes